packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -37,10 +40,208 @@ enum _ScaffoldSlot {
   statusBar,
 }
 
-/// Geometry information for scaffold components.
+/// Responsible for placing the [FloatingActionButton] in the [Scaffold].
```

Comment:
Maybe "Interface for objects that place..."?

Reply:
Done